sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/AbstractSentryWebFilter.java

```diff
@@ -34,16 +34,17 @@
 @ApiStatus.Experimental
 public abstract class AbstractSentryWebFilter implements WebFilter {
   private final @NotNull SentryRequestResolver sentryRequestResolver;
-  public static final String SENTRY_HUB_KEY = "sentry-hub";
+  public static final String SENTRY_SCOPES_KEY = "sentry-scopes";
+  @Deprecated public static final String SENTRY_HUB_KEY = SENTRY_SCOPES_KEY;
   private static final String TRANSACTION_OP = "http.server";
 
-  public AbstractSentryWebFilter(final @NotNull IHub hub) {
-    Objects.requireNonNull(hub, "hub is required");
-    this.sentryRequestResolver = new SentryRequestResolver(hub);
+  public AbstractSentryWebFilter(final @NotNull IScopes scopes) {
+    Objects.requireNonNull(scopes, "scopes are required");
+    this.sentryRequestResolver = new SentryRequestResolver(scopes);
   }
 
   protected @Nullable ITransaction maybeStartTransaction(
-      final @NotNull IHub requestHub, final @NotNull ServerHttpRequest request) {
+      final @NotNull IScopes requestHub, final @NotNull ServerHttpRequest request) {
```

Comment:
Should we rename this to something like `requestScopes`?

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366